### ISSUE TITLE
Remove depreciated temperature label

### ISSUE THREAD

**Issue by rhinoella** [trisyoungs/duq#1843]
Configuration tab still contains depreciated temperature indicator

**Screenshots**
![image](https://github.com/disorderedmaterials/dissolve/assets/101950441/ebd0869c-4403-481e-8a65-a6f4ba88f026)

**Operating System:**
 - OS: OSX
 - OS version / flavour:  Ventura

**Comment by trisyoungs** [comment]
@rhinoella You are absolutely correct here, but this is more "unexpected behaviour" than it is a bug. The `TemperatureProcedureNode` is deprecated, and only sets the temperature of the configuration when generating the configuration. Hence, simply changing the value in the node won't change the value in the top-left of the `ConfigurationTab` (as you have observed) until you "Generate" the configuration again.

There is no easy way to account for this discrepancy until `TemperatureProcedureNode` is fully removed, so to mitigate the effects we should remove references to it from the documentation, stop them being added programatically in the code when adding configurations, and refer the user to the main menu item instead.
